ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -37,7 +37,7 @@
 #include <nnfw.h>
 
 /** backends supported by nnfw */
-#define NNFW_CPU_BACKEND  "cpu"
+#define NNFW_CPU_BACKEND  "cpu;bcq"
```

Comment:
This does change the default backend but I recommend to create a NNFW_CPU_BCQ_BACKEND here and use it.
Further, register a ACCL_CPU_BCQ_STR and ACCL_CPU_BCQ at `gst/nnstreamer/include/nnstreamer_plugin_api_filter.h` and then update the auto/default backend at line 55 to 60 for various architectures.

Reply:
Until we get it updated, let's use either:
```
#define NNFW_CPU_BACKEND  "cpu;bcq"
```
or
```
#define NNFW_DEFAULT_BACKEND NNFW_CPU_BACKEND ";bcq"
```
Let's not touch the API related headers at this moment (release imminent)

Reply:
Besides, as long as bcq enabled just yet another algorithm using CPU instructions, having "cpu;bcq" as CPU_BACKEND  isn't terribly wrong.